CacheMeIfYouCan/CacheMeIfYouCan/CacheTypes/FileSystemDataCache.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  FilesystemDataCache.swift
+//  CacheMeIfYouCan
+//
+//  Created by Ellen Shapiro on 12/10/18.
+//  Copyright © 2018 Bakken & Baeck. All rights reserved.
+//
+
+import Foundation
+
+open class FileSystemDataCache<T: DataConvertible>: Cache {
+    
+    public typealias StoredType = T
+    
+    let folderURL: URL
```

Comment:
Nit pick that applies to more location that just this.

You're using the nomenclature `folder` and `directory` - maybe not an issue with autocomplete, but there are benefits of using the same term throughout. Framework uses `directory` could be worth removing all references to  `folder`

Reply:
yeah good point.